tests/integration/test_plan/test_plan_templates/test__create_test_plan_template.py

```diff
@@ -0,0 +1,132 @@
+from typing import List
+import pytest
+
+from nisystemlink.clients.core._http_configuration import HttpConfiguration
+from nisystemlink.clients.test_plan.models._execution_definition import ManualExecutionDefinition
+from nisystemlink.clients.test_plan.test_plan_templates._test_plan_templates_client import TestPlanTemplateClient
+from nisystemlink.clients.test_plan.test_plan_templates.models._create_test_plan_templates import CreateTestPlanTemplateResponse
+from nisystemlink.clients.test_plan.test_plan_templates.models._delete_test_plan_templates import DeleteTestPlanTemplates
+from nisystemlink.clients.test_plan.test_plan_templates.models._query_test_plan_templates import QueryTestPlanTemplatesRequestBody, QueryTestPlanTemplatesResponse
+from nisystemlink.clients.test_plan.test_plan_templates.models._test_plan_templates import TestPlanTemplateBase
+
+
+@pytest.fixture(scope="class")
+def create_test_plan_template() -> List[TestPlanTemplateBase]:
+    """Fixture to create test plan template."""
+
+    testPlanTemplates = [
+        TestPlanTemplateBase(
+            name = "Sample testplan template",
+            templateGroup = "sample template group",
+            workspace = "33eba2fe-fe42-48a1-a47f-a6669479a8aa",
+            executionActions=[
+                ManualExecutionDefinition(
+                    action="TEST",
+                    type="MANUAl"
+                )
+            ]
+        )
+    ]
+
+    return testPlanTemplates
+
+@pytest.fixture(scope="class")
+def client(enterprise_config: HttpConfiguration) -> TestPlanTemplateClient:
+    """Fixture to create a TestPlanTemplateClient instance."""
+
+    return TestPlanTemplateClient(enterprise_config)
+
+@pytest.mark.integration
+@pytest.mark.enterprise
+class TestTestPlanTemplate:
```

Comment:
```suggestion
class TestPlanTemplateTests:
```
I guess, pytest only check for file names start with `test_`, not with the class name.